store/precomputed_key/s3/cli.go

```diff
@@ -35,9 +36,16 @@ func CLIFlags(envPrefix, category string) []cli.Flag {
 			EnvVars:  withEnvPrefix(envPrefix, "S3_ENDPOINT"),
 			Category: category,
 		},
+		&cli.BoolFlag{
+			Name:     EnableTLSFlagName,
+			Usage:    "enable TLS connection to S3 endpoint",
+			Value:    false,
+			EnvVars:  withEnvPrefix(envPrefix, "S3_ENABLE_TLS"),
```

Comment:
WithEnvPrwrix adds the S3 prefix. Should be removed. Seems like we forgot to remove it on some other flags as well. Do you mind changing those as well?